### ISSUE TITLE
NTP for Ubuntu 16.04 and above

### ISSUE THREAD

**Issue by kevinwheeler** [imthenachoman/How-To-Secure-A-Linux-Server#86]
Two separate questions.

1) Is the NTP setup necessary for Ubuntu versions above 16.04? Or is this no longer needed due to the fact that these versions of Ubuntu come with timesyncd? (sources: Corey Goldberg's comment on this answer https://askubuntu.com/a/641160 and these Digital Ocean articles: https://www.digitalocean.com/community/tutorials/how-to-set-up-time-synchronization-on-ubuntu-20-04 - https://www.digitalocean.com/community/tutorials/how-to-set-up-time-synchronization-on-ubuntu-18-04 - https://www.digitalocean.com/community/tutorials/how-to-set-up-time-synchronization-on-ubuntu-16-04 )

2) If NTP setup is necessary, do we need to disable timesyncd as per the following quote: "Before installing ntpd, you need to turn off timesyncd in order to prevent the two services from conflicting with one another." from this article https://www.digitalocean.com/community/tutorials/how-to-set-up-time-synchronization-on-ubuntu-16-04 ?

**Comment by imthenachoman** [comment]
I don't think it is necessary anymore. I lost my Linux server a while ago (long story) and I haven't replaced it yet so I've been out of the loop with what is necessary. I'll try to find time to research and update this guide. Or if you want to submit a PR I would be happy to merge it.

**Comment by kiv57** [comment]
+1 : it seems that for an NTP client purpose only, the `systemd-timesyncd` does a minimalist but sufficient job, except for use cases requiring greater precision maybe ?
[a stackexchange question comparing NTP and timesyncd](https://unix.stackexchange.com/questions/635088/the-new-way-to-synchronize-with-time-is-systemd-timesyncd-what-is-the-replaceme)

**Comment by imthenachoman** [comment]
Yeah. I'm thinking maybe this is not needed anymore but I don't have a server anymore that I can confirm on. I run Debian + KDE on my daily driver and it came with NTP but I'm not sure if that's cause I'm using it as a personal computer, and not a server, so it installed it.
